### ISSUE TITLE
Importing playlist creates empty playlist

### ISSUE THREAD

**Issue by barlowl** [TeamPiped/Piped#2617]
### Official Instance

- [X] The bug is reproducible on the [official hosted instance](http://piped.video/), or is API-related.

### Describe the bug

When I attempt to import a playlist (a .csv from YouTube's official takeout data), I get an untitled, empty playlist with 0 videos.

### To Reproduce

1. Open an instance and click on "Playlists"
2. Select "Import from JSON/CSV"
3. Select the .csv file and select open
4. Upon loading, refresh the page
5. Blank playlist appears

### Expected behavior

I would expect the playlist with the correct amount of videos and the correct title to appear.

### Browser, and OS with Version.

Firefox ESR 102.12.0 on Gentoo GNU/Linux 6.1.31-gentoo-dist. I used vanilla Firefox.

**Comment by Bnyro** [comment]
With or without account?

**Comment by barlowl** [comment]
When I tried logged in the playlist simply didn't appear at all as if I had never imported the file.

**Comment by OMEGA-END** [comment]
i had the same problem, in my case the playlist had some unavailable (private or deleted) videos. removing them on youtube and exporting again should solve the problem.

**Comment by barlowl** [comment]
Only some of the playlists I was trying to export had unavailable videos, but this blank playlist issue occurred on all my playlists, not just the ones with unavailable videos. I've since removed all the unavailable videos and reexported and the problem still persists.

I've tried different instances, changing my environment, and using JSON files and nothing has worked so far. Because my playlists are so long, I don't want to have to manually import them into Piped.
